content/en/docs/papers/fsep/_index.md

```diff
@@ -0,0 +1,218 @@
+| 2.5 | Remove Instance from deny list: It should be possible for an administrator to remove instances where deemed appropriate. These changes only apply to local deny lists and will not affect subscribed deny lists.  | P1 |
+
+### Moderation Data
+
+Moderation capabilities will be expanded to provide more context and information to make informed decisions. 
+
+| ID | FEATURE | PRIORITY |
+|:--|:--|:--|
+| 3.1 | It should be possible for admins to see how many of their users’ followers / following would be impacted by a deny list. For example: *“When you subscribe to [The Bad Space], [Number] of your users will no longer be able to communicate with [number] followers and [number] following.”*| P1 |
+| 3.2 | Check for updates: It should be possible for an admin to manually update any of the deny lists that they are subscribed to outside of the automated update time. Administrators should be prompted with moderation information and be able to confirm the status before accepting an update.  | P1 |
+| 3.3 | Administrators should be able to see a log of updates (temporal diffs) for sites on the deny list.  | P1 |
+
+## Non-Functional Requirements
+
+| ID | FEATURE | PRIORITY |
+|:--|:--|:--|
+| 4.1 | The update time should apply jitter so that all instances don’t try to update simultaneously. | P2 |
+| 4.2 | Users should be able to add up to 100 default deny lists. For v1, there will only be one default. But the UI should consider that future updates will add more options. | P3 |
+| 4.3 | Inclusive language: The terminology should be “deny list.” | P3 |
+
+## Design
+
+
+### Onboarding UI
+
+The onboarding process flow for platforms on the fediverse involves inputting various identifying pieces of information such as email, username, and password for the administration account, deciding on multiple features of the chosen platform, and confirming that these details are correct. The following sample UI mocks are non-implementation specific, so they should be applicable across several fediverse applications ( Mastodon, MissKey, CalcKey, etc.).
+
+An additional UI component can be added to this flow to inform the administrator of the features detailed in the Onboarding section.
+
+The first panel of the new component will describe what The Bad Space is, what it does, and how it will be used. It will also contain an element to confirm approval by the administrator to use The Bad Space’s feature set in the platform's experience. If the administrator declines, the flow will continue with the default onboarding process.
+
+![ONBOARDING BLOCKLIST PANEL](Blocklist%20Panel%20One@1x.png)
+
+Upon consenting to the system to use the additional features, a second panel will be revealed to ask if they would like the dynamic blocklist automatically imported from The Bad Space to populate the system's blocklist with the option to check for updates.     
```

Comment:
The word "consenting" here is a bit odd — I know what you mean, but maybe "connecting" or "integrating" might be better?

Reply:
Hm. The paradigm for the connection between The Bad Space and other platforms is a symbiotic relationship, so the word choice is specific.

I'm going for less techy and more accessible language.